lib/rex/proto/apache_j_p.rb

```diff
@@ -0,0 +1,127 @@
+# -*- coding: binary -*-
+
+require 'bindata'
+
+# @see: https://tomcat.apache.org/connectors-doc/ajp/ajpv13a.html
+module Rex::Proto::ApacheJP
```

Comment:
I'm open to updating it to consolidate the code if we think that's worthwhile but I think it'd make sense to do that in a separate change set to avoid blocking this one.

Reply:
Agreed! 💯